dht/token.hh

```diff
@@ -122,6 +123,8 @@ public:
      */
     static std::map<token, float> describe_ownership(const std::vector<token>& sorted_tokens);
 
+    static data_type get_token_validator();
```

Comment:
Why make it a static method instead of a normal function?

Reply:
As Avi answered on one of your other comments about free functions, this is about discoverability. All those functions are token related so they are grouped under token namespace. I will resolve all other comments regarding free functions because they have the same answer.